src/main/kotlin/no/nav/familie/klage/behandlingsstatistikk/PatchBehandlingsstatistikkController.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.familie.klage.behandlingsstatistikk
+
+import no.nav.familie.klage.behandling.BehandlingService
+import no.nav.familie.kontrakter.felles.klage.BehandlingResultat
+import no.nav.familie.prosessering.domene.Status
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import no.nav.security.token.support.core.api.Unprotected
+import org.slf4j.LoggerFactory
+import org.springframework.data.domain.Pageable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDate
+import java.util.UUID
+
+@RestController
+@RequestMapping("/api/patch-statistikk")
+@Unprotected
+class PatchStatistikkController(
+    private val behandlingService: BehandlingService,
+    private val taskRepository: TaskRepository
+) {
+    private val triggerTid = LocalDate.of(2022, 11, 11)
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    @PostMapping("/")
+    fun patchTidligereBehandlingResultater() {
+        val tasker = hentOgFiltrerTaskerPåTriggertid()
+        tasker.forEach { task ->
+            val behandlingId = task.getProperty("behandlingId")
+            val behandling = behandlingService.hentBehandling(UUID.fromString(behandlingId))
+            if (behandling.resultat == BehandlingResultat.IKKE_MEDHOLD) {
+                if (sendtTilKATaskErOpprettet(behandlingId, tasker)) {
+                    logger.info("Det finnes allerede en task med behandlingId:$behandlingId som har status FERDIG av typen SENDT_TIL_KA")
+                    return@forEach
```

Comment:
Jeg vet noen i teamet ikke liker slike returns, for meg spiller det ikke så stor rolle og spesiellt ikke her. Med en else trenger du ikke return

Reply:
Ja, har ikke noe å si for meg heller, så lar return ligge hvis ingen roper :)